cli/package.json

```diff
@@ -94,5 +94,8 @@
     "type-fest": "^3.7.0",
     "typescript": "^5.4.2",
     "zod": "^3.23.3"
-  }
+  },
+  "trustedDependencies": [
+    "es5-ext"
```

Comment:
????

Reply:
idk, it was automatically added by my package manager

Reply:
will remove